plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -949,6 +950,24 @@ protected WriteMapping legacyToWriteMapping(@SuppressWarnings("unused") Connecto
         throw new TrinoException(NOT_SUPPORTED, "Unsupported column type: " + type.getDisplayName());
     }
 
+    protected boolean preventTextualTypeAggregationPushdown(List<AggregateFunction> aggregates, List<List<ColumnHandle>> groupingSets)
+    {
+        // The remote database may be case-insensitive or sort textual types differently than Trino.
```

Comment:
> remote database may be case-insensitive

nit: Use "Remote database can be case insensitive" words, so that similar places in the code are searchable.